tests/integration/instrumentation/__init__.py

```diff
@@ -1,15 +1,15 @@
 from typing import Dict, Optional
 
+import requests as http_requests
 from docarray import DocumentArray
 from opentelemetry.context.context import Context
+from prometheus_api_client import PrometheusConnect
 
 from jina import Executor, requests
 
 
 def get_traces(service):
-    import requests
-
-    response = requests.get(f'http://localhost:16686/api/traces?service={service}')
+    response = http_requests.get(f'http://localhost:16686/api/traces?service={service}')
```

Comment:
is there a way to make this `port` random?

Reply:
I can try. I have to export the random port to the docker-compose as environment variable or docker args if possible.